idea/testData/intentions/replaceSingleLineLetIntention/letWithThisShortBinaryExpression.kt.after

```diff
@@ -0,0 +1,6 @@
+// WITH_RUNTIME
+// IS_APPLICABLE: true
+
+fun Int.foo() {
+    +1
```

Comment:
This transformation does not look correct for me. I would expect `this + 1` as a result here,